src/imageio/storage/piwigo.c

```diff
@@ -1265,6 +1270,14 @@ int store(dt_imageio_module_storage_t *self,
   dt_storage_piwigo_gui_data_t *ui = self->gui_data;
   dt_storage_piwigo_params_t *p = (dt_storage_piwigo_params_t *)sdata;
 
+  if(p->api == NULL)
+  {
+    dt_print(DT_DEBUG_ALWAYS,
+             "[imageio_storage_piwigo] not logged in to piwigo server!\n");
+    dt_control_log(_("Not logged in to piwigo server!"));
```

Comment:
The original strings in the UI are written without capitalization at the beginning of sentences, so for consistency we should not break this convention in the new strings.
But the proper name of the program/service must be written without changing the case: Piwigo